site/cypress/e2e/Cypress-TAGrading/clearing_version_conflicts.spec.js

```diff
@@ -38,4 +38,47 @@ describe('Test cases for checking the clear version conflicts button in the TA g
         cy.reload();
         cy.get('[data-testid="version-warning"]').should('exist');
     });
+
+    it('Clicking "save" on a component with a version conflict should resolve the conflict for that component', () => {
+        cy.login('instructor');
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'grade?who_id=K8jI3q4qpdCc1jw&sort=id&direction=ASC&gradeable_version=1']);
+        cy.get('#grading_rubric_btn').click();
+
+        // wait until page is fully loaded
+        cy.get('#component-list').children().should('have.length.least', 1);
```

Comment:
Could you use testids for the elements tested in this PR?

Reply:
My bad I thought I changed them already